RadixWallet/Features/AssetsFeature/Components/PoolUnitsList/Components/LSUResource/LSUStake+View.swift

```diff
@@ -159,7 +151,7 @@ extension View {
 
 extension LSUStake.State {
 	var viewState: LSUStake.ViewState {
-		.init(id: stake.validatorAddress, content: stakeDetails.map { details in
+		LSUStake.ViewState(id: stake.validatorAddress, content: stakeDetails.map { details in
```

Comment:
Mm good, I think if we change ALL our `.init` to use explicit types instead, that ought to have a pretty big impact on type-checking time?